steps/run-android-instrumentation-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,136 @@
+title: Run instrumentation tests
+summary: |
+  Run your Android UI tests with [emulator.wtf](https://emulator.wtf).
+description: |
+  Run your Android UI tests with [emulator.wtf](https://emulator.wtf).
+
+  Emulator.wtf is an Android cloud emulator laser-focused on performance to deliver quick feedback to your PRs.
+
+  It's like Firebase Test Lab on steroids (2-4x speedup, depending on your specific setup)
+website: https://emulator.wtf
+source_code_url: https://github.com/emulator-wtf/run-tests-bitrise
+support_url: https://github.com/emulator-wtf/run-tests-bitrise/issues
+published_at: 2021-11-28T17:10:15.656951+02:00
+source:
+  git: https://github.com/emulator-wtf/run-tests-bitrise.git
+  commit: b4e5502f25bbec6c9b1db6f7f519b0361b293ad3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
That script downloads a binary from external server at: https://github.com/emulator-wtf/run-tests-bitrise/blob/main/step.sh#L4
That is against step development guideline:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#do-not-use-submodules-or-require-any-other-resource-downloaded-on-demand